modules/ding_event/ding_event.module

```diff
@@ -94,6 +94,18 @@ function ding_event_theme() {
       ),
       'template' => 'templates/views-exposed-form-widget',
     ),
+    'views_exposed_form_widgets_title' => array(
+      'variables' => array(
+        'element' => NULL,
+      ),
+      'template' => 'templates/views-exposed-form-widget--title',
+    ),
+    'views_exposed_form_widgets_date_inputs' => array(
+      'variables' => array(
+        'element' => NULL,
+      ),
+      'template' => 'templates/views-exposed-form-widget--date-inputs',
+    ),
```

Comment:
https://inlead.atlassian.net/browse/EASYOPAC-1115

Reply:
I can't read from this issue, what this means. Can you?

Reply:
Custom templating for exposed filters - https://github.com/easySuite/ding2/pull/581/files

Reply:
Well I can find commits and read them. But that does not tell me what it does, how it will look, etc.

Reply:
Explain what this change does and why its needed.

Reply:
Formats the date inputs on /arrangementer page in new theme:
![image](https://user-images.githubusercontent.com/574498/99975221-a3291180-2daa-11eb-8221-dfaad1a5e67d.png)

Reply:
This is the same format as there currently is in the main distro:
![image](https://user-images.githubusercontent.com/287648/99978567-65c28500-2da6-11eb-81ca-fdefec08060e.png)

Reply:
Great, so I assume this is not needed?

Reply:
I would assume so too.